plugins/tabletools/plugin.js

```diff
@@ -305,55 +310,55 @@
 		return addedCells;
 	}
 
-	function deleteColumns( selection ) {
-		function processSelection( selection ) {
-			// If selection leak to next td/th cell, then preserve it in previous cell.
+	function preserveSelection( selection ) {
```

Comment:
It'd be nice to move this helper function after all functions that use it. It should increase the readability of the code.